### PR TITLE
Adding description and eligibility (#36)

### PR DESCRIPTION
**Pull request by James-S-Santangelo**
Adding submission description and eligibility for JOSE (#36).

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -72,6 +72,11 @@ This is an example citation [@figshare_archive].
 
 Figures can be included like this: ![Fidgit deposited in figshare.](figshare_article.png)
 
+# Description and eligibility
+
+We present a stand-alone module designed for simultaneously teaching statistical and programming skills in a participatory live-coding format to upper year undergraduate biology students. Our module has 16 self-contained lectures, with topics ranging from data cleaning, manipulation, and visualization, to the statistical analysis of biological data and reproducibility through GitHub. Our module additionally includes eight assignments that were distributed to students throughout the term to assess learning and understanding of course material. All materials were developed and taught using R markdown, although no prior knowledge of the R programming language is assumed prior to taking the course. Course content is licensed under the CC-BY 4.0 licence while code is licensed under MIT. The open-source and stand-alone nature of our computational and statistical learning module makes this material ideally suited to adoption by other instructors and an excellent contribution to JOSE.
```
Comment (mbonsma) [inline paper.md]: * `simultaneously teaching statistical skills and programming skills`
* no comma after `self-contained lectures`

I'm not sure that this emphasizes enough that lectures could be used independently - maybe in the last sentence there's a synonym to 'modular' or 'stand-alone' that would hammer it home? But we do say it in the Statement of Need too.

**Review by mbonsma — CHANGES_REQUESTED** [review]

File: paper.md (hunk 1 above)
Comment (lwjohnst86) [reply paper.md]: "16 primarily self-contained lectures"

Comment (lwjohnst86) [reply paper.md]: "R Markdown"

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]
Looking good! Minor edits

File: paper.md [hunk 2]
```diff
@@ -72,6 +72,11 @@ This is an example citation [@figshare_archive].
 
 Figures can be included like this: ![Fidgit deposited in figshare.](figshare_article.png)
 
+# Description and eligibility
+
+We present a stand-alone module designed for simultaneously teaching statistical skills and programming skills in a participatory live-coding format to upper year undergraduate biology students. Our module has 16 primarily self-contained lectures, with topics ranging from data cleaning manipulation, and visualization, to the statistical analysis of biological data and reproducibility through GitHub. Our module additionally includes eight assignments that were distributed to students throughout the term to assess learning and understanding of course material. All materials were developed and taught using R Markdown, although no prior knowledge of the R programming language is assumed prior to taking the course. Course content is licensed under the CC-BY 4.0 licence while code is licensed under MIT. The open-source and stand-alone nature of our computational and statistical learning module makes this material ideally suited to adoption by other instructors and an excellent contribution to JOSE.
```
Comment (mbonsma) [inline paper.md]: `data cleaning, manipulation`

**Review by mbonsma — APPROVED** [review]

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**